ovirt-web-ui.spec.in

```diff
@@ -20,19 +20,22 @@ Source0:        https://github.com/oVirt/ovirt-web-ui/archive/%{source_basename}
 
 ExclusiveArch:  x86_64
 
-BuildRequires: ovirt-engine-nodejs = 6.9.1
-# Keep ovirt-engine-nodejs-modules at particular version unless tested on higher
-BuildRequires: ovirt-engine-nodejs-modules = 0.0.17
+# Keep ovirt-engine-{nodejs|nodejs-modules|yarn} at particular version unless tested on higher
+# Please keep in sync with the automation/*.req files
+BuildRequires: ovirt-engine-nodejs=6.9.4
+BuildRequires: ovirt-engine-nodejs-modules=1.0.3
+BuildRequires: ovirt-engine-yarn=0.19.1
```

Comment:
I can be mistaken, yes. I tried locally with `mock_runner.sh`, and for it failed to install the packages. I did something wrong, probably.